tee-worker/ts-tests/identity.test.ts

```diff
@@ -116,10 +118,10 @@ describeLitentry('Test Identity', 0, (context) => {
         // the main address should be already inside the IDGraph
         const main_identity = await buildIdentityHelper(
             u8aToHex(context.substrateWallet.alice.addressRaw),
-            'LitentryRococo',
+            process.env.NODE_ENV === 'local' ? 'TestNet' : 'LitentryRococo',
```

Comment:
Should we rather centralize this somewhere rather than having these checks and magic constants scattered all over the place? :thinking:

Reply:
fixed!